cmd/commands/cli/command.go

```diff
@@ -21,29 +21,61 @@ import (
 	"fmt"
 	"io"
 	"log"
+	"path/filepath"
 	"strconv"
 	"strings"
 
 	"github.com/chzyer/readline"
+	"github.com/mysteriumnetwork/node/cmd"
 	"github.com/mysteriumnetwork/node/metadata"
 	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
 	"github.com/mysteriumnetwork/node/tequilapi/endpoints"
 	"github.com/mysteriumnetwork/node/utils"
+	"github.com/urfave/cli"
 )
 
-// NewCommand constructs CLI based with possibility to control quiting
-func NewCommand(
-	historyFile string,
-	tequilapi *tequilapi_client.Client,
-) *Command {
-	return &Command{
-		historyFile: historyFile,
-		tequilapi:   tequilapi,
+const cliCommandName = "cli"
+
+// NewCommand constructs CLI based Mysterium UI with possibility to control quiting
+func NewCommand() *cli.Command {
+	var di cmd.Dependencies
+
+	return &cli.Command{
+		Name:  cliCommandName,
+		Usage: "Starts a CLI client with a Tequilapi",
+		Action: func(ctx *cli.Context) error {
+			nodeOptions := cmd.ParseFlagsNode(ctx)
+			if err := di.Bootstrap(nodeOptions); err != nil {
+				return err
+			}
+			cmd.RegisterSignalCallback(utils.SoftKiller(di.Node.Kill))
+
+			errorChannel := make(chan error)
+			go func() {
+				if err := di.Node.Start(); err != nil {
```

Comment:
`Node` starting/stoping logic is duplicated now. Maybe we can wrap with like: `NewDaemonCommand(command_cli.NewCommand()) * cli.Command`

Reply:
Extracted `daemon` to the separate function.